src/components/NumberInput/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ChangeEvent, FC } from 'react';
+import styled from 'styled-components';
+
+const NumberInputContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
+  width: 100%;
+`;
+
+const StyledLabel = styled.label`
+  display: block;
+  font-size: ${p => p.theme.fontSizeLl};
+`;
+
+const StyledInput = styled.input`
+  padding: 10px;
+  margin-top: 8px;
+  border-radius: 4px;
+  border: 1px solid ${p => p.theme.colorTextMedium};
+  color: ${p => p.theme.colorTextDark};
+  &::selection {
+    background: ${p => p.theme.colorPrimary};
+    color: white;
+  }
+  &:focus {
+    outline: none;
+    box-shadow: 0 0 0 3px rgba(0, 0, 0, 0.1);
+  }
+`;
+
+export interface NumberInputType {
+  id: string;
+  label: string;
+  min?: number;
+  max?: number;
+  step?: number;
+  placeholder?: string;
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const NumberInput: FC<NumberInputType> = ({
+  id,
+  label,
+  min = 0,
```

Comment:
@dnsos Shouldn't this be min 1 not 0? We don't want user to add a 0 watering?

Reply:
You're right, I've disallowed values <= 0 now ([9f08c6e](https://github.com/technologiestiftung/giessdenkiez-de/pull/441/commits/9f08c6e6be48902f3dc90fd331f03f4c23e70979)). However I wouldn't modify 0 as the default min value of the `NumberInput`. This is because the users should consciously think about the amount and not submit the value 1 by accident.

Reply:
I guess that 0 forces the user to make a decision and avoid inadvertent watering right?